lib/server.js

```diff
@@ -0,0 +1,34 @@
+const {
+  AUTH_HEADER,
+  JSON_WEB_TOKEN,
+  NPM_SECRET,
+  DEEP_HEADER_AUTHORIZATION,
+  DEEP_HEADER_SET_COOKIE,
+  REWRITE_REQUEST,
+  REWRITE_RESPONSE,
+} = require('./matchers')
+
+const {
+  redactUrlMatcher,
+  redactUrlPasswordMatcher,
+  redactMatchers,
+} = require('./utils')
+
+const { deepMap } = require('./deep-map')
```

Comment:
I wanted to point out that this is a good idea to only import this in `lib/server` and not put it in `utils.js`.

One thing we've been trying to stay aware of on the CLI is not importing more code than necessary on startup, which is not really a concern we have on the registry.